src/consensus/aft/raft.h

```diff
@@ -2823,7 +2894,7 @@ namespace aft
         LOG_INFO_FMT("Removed raft self {}", state->my_node_id);
         if (replica_state == kv::ReplicaState::Leader)
         {
-          become_retired();
+          // become_retired();
```

Comment:
Remove?

Reply:
Yes, thank you, I'd forgotten about this. This whole path is no longer necessary.